harness/setup.py

```diff
@@ -29,8 +29,10 @@
         "filelock",
         "google-cloud-storage",
         "hdfs>=2.2.2",
+        "jsonschema",
         "lomond>=0.3.3",
         "pathspec>=0.6.0",
+        "pyyaml",
```

Comment:
where is this from?  Don't we always use ruamel.yaml?

Reply:
can you just fix those instead?  The `pyyaml` library has issues, and we set up a better solution that is much more widely used:

```
rb@bearasaurusrex ~/code/determined/harness/determined (git)-[master]
% git grep '^import yaml'
cli/job.py:import yaml
deploy/aws/gen_vcpu_mapping.py:import yaml
deploy/aws/preflight.py:import yaml
```

vs 

```
rb@bearasaurusrex ~/code/determined/harness/determined (git)-[master]
% git grep '^import yaml'
cli/job.py:import yaml
deploy/aws/gen_vcpu_mapping.py:import yaml
deploy/aws/preflight.py:import yaml
```